muleauth-disovery.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Set up different auth resource files for traceability and discovery for ease of demo as our agents currently use two different accounts for the intended demo, will combine them into one post demo.